NEWS.md

```diff
@@ -46,7 +46,7 @@ Allowing documenting types created with `@with_kw`
 Added `@pack` and `@unpack` macros:
 
 ```julia
-type A
+mutable struct A
```

Comment:
bad bot

Reply:
I think the NEWS.md file should be skipped as it should preserve its historical context.